src/cascadia/TerminalSettingsModel/JsonUtils.h

```diff
@@ -177,13 +175,27 @@ namespace Microsoft::Terminal::Settings::Model::JsonUtils
         // Leverage the wstring converter's validation
         winrt::hstring FromJson(const Json::Value& json)
         {
+            if (json.isNull())
+            {
+                return winrt::hstring{};
+            }
             return winrt::hstring{ til::u8u16(Detail::GetStringView(json)) };
         }
 
         Json::Value ToJson(const winrt::hstring& val)
         {
+            if (val == winrt::hstring{})
+            {
+                return Json::Value::nullSingleton();
+            }

```

Comment:
_Interesting_. So we store `null` as `""` but write it out as `null`. I like that.

Reply:
I mean, this is extremely exactly how hstring works. We've talked about this, I think: _THERE IS NO EMPTY HSTRING_. There is an hstring that contains text, or an hstring that is null. There is no `""` hstring.